actions/code.js

```diff
@@ -8,10 +8,10 @@ const request = require('co-request');
 function wait(timeout) {
   return new Promise((ok) => {
     setTimeout(() => {
-      this.logger.debug('Done wait');
+      this.logger.info('Done wait');
```

Comment:
Why changing this? This is existing behavior and out of scope of this task

Reply:
Igor asked for this in general components